signer/src/error.rs

```diff
@@ -34,26 +40,19 @@ pub enum Error {
     #[error("invalid signing request")]
     InvalidSigningOperation,
 
-    /// No mock transaction was found after DKG successfully completed. Spending
-    /// a signer UTXO locked by the new aggregate key could not be verified.
-    #[error("no mock transaction found when attempting to sign")]
-    MissingMockTransaction,
+    /// The DKG verification state machine is in an end-state.
+    #[error("DKG verification state machine is in an end-state and cannot be used for the requested operation: {0}")]
+    DkgVerificationEnded(PublicKeyXOnly, Box<dkg::verification::State>),
 
     /// The rotate-key frost verification signing round failed for the aggregate
```

Comment:
Since you renamed the error text, maybe align the comment too?

Reply:
[75c78bc](https://github.com/stacks-network/sbtc/pull/1324/commits/75c78bc3c46868836857bc3750671ccbec4c83c3)